fedoracommunity/widgets/package/bugs.py

```diff
@@ -43,21 +43,29 @@ def prepare(self):
 
         releases = []
         self.filters = {'package': self.package}
-        pkgdb = get_connector('pkgdb')
-        collections = pkgdb.get_collection_table(active_only=True)
-
-        for id, collection in collections.items():
-            name = collection['name']
+        bodhi = get_connector('bodhi')
+        releases_obj = bodhi.call('releases', {})
+        total_pages = releases_obj['pages']
+        collections = releases_obj['releases']
+        if(total_pages > 1):
```

Comment:
This is the same code has above. Can you move this code and the other code into a function to avoid code duplication?